accounts/static/accounts/js/register.js

```diff
@@ -0,0 +1,23 @@
+$(function () {
+    $("input[open_modal=\"true\"]").on("change", function () {
```

Comment:
You can have $("input[open_modal=\"true\"]") as variable at the top so it doesn't search for it every time. Make the performance better.